data/421/300/519/421300519.geojson

```diff
@@ -35,35 +36,35 @@
         "gp:id":"12724319"
     },
     "wof:country":"FR",
-    "wof:geomhash":"fd267690b02021ca6cbfbaeb845e0d48",
+    "wof:geomhash":"f380a932d9fd515d572c6176e3b2126c",
     "wof:hierarchy":[
         {
             "continent_id":102191581,
             "country_id":85633147,
             "county_id":102067427,
             "empire_id":136253037,
-            "localadmin_id":404421605,
+            "localadmin_id":404359359,
```

Comment:
Similar case here. We're suggesting this postalcode

https://spelunker.whosonfirst.org/id/421300519/

is now parented by locaadmin:

https://spelunker.whosonfirst.org/id/404359359/

When in reality, it looks like most of these are in the same postalcode shape:

https://spelunker.whosonfirst.org/id/102066429/descendants/?exclude=nullisland&placetype=locality

Is that true? Again, these edits look fine.. I just want to understand if we should file a follow up to untangle multiple-hierarchies in France postalcodes.

Reply:
Yes, that's totally right, a postalcode (not all postalcodes) can contains many localadmins.

So they may use multiple hierarchies. The shape of a postal code should fit perfectly the union of their localadmins. (In WOF, localadmins are more accurate than postalcodes)